internal/tools/aws/ec2.go

```diff
@@ -163,3 +167,29 @@ func GetSecurityGroupsWithFilters(filters []*ec2.Filter) ([]*ec2.SecurityGroup,
 
 	return sgOutput.SecurityGroups, nil
 }
+
+// IsValidAMI check if the provided AMI exists
+func (a *Client) IsValidAMI(AMIImage string) (bool, error) {
+	svc := ec2.New(session.New(), &aws.Config{
+		Region: aws.String(DefaultAWSRegion),
+	})
+
+	describeImageInput := &ec2.DescribeImagesInput{
+		Filters: []*ec2.Filter{
+			{
+				Name:   aws.String("image-id"),
+				Values: []*string{aws.String(AMIImage)},
+			},
+		},
+	}
+
+	out, err := a.api.describeImages(svc, describeImageInput)
+	if err != nil {
+		return false, err
+	}
+	if len(out.Images) == 0 {
+		return false, err
```

Comment:
`err` would be nil here. We should probably return something else or `nil` directly.